### PR TITLE
Added scripts to create local client ws

### PR DESCRIPTION
**Pull request by BorjaOuterelo**
Added scripts to create a client WS in the host machine.
Added micro-ROS-demos repository.
Also unified create_client_ws.sh script with the rest of the create_* scripts.

### CODE REVIEW

**Comment by iluetkeb** [comment]
Scripts are looking good. Could you please add a CI configuration that uses them to make sure everything works?

You can add it to .gitlab-ci.yml, hopefully the other entries give you an idea of how to do it.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -47,6 +48,36 @@ agent_ws:
...
+    - test -f install/lib/rad0_actuator_c/rad0_actuator_c
+    - test -f install/lib/rad0_altitude_sensor_c/rad0_altitude_sensor_c
+    - test -f install/lib/rad0_control_cpp/rad0_control_cpp
+    - test -f install/lib/rad0_display_c/rad0_display_c
+    - test -f install/lib/string_publisher_c/string_publisher_c
+    - test -f install/lib/string_publisher_cpp/string_publisher_cpp
+    - test -f install/lib/string_subscriber_c/string_subscriber_c
+    - test -f install/lib/string_subscriber_cpp/string_subscriber_cpp
+  only:
+    - master
```
Comment (iluetkeb) [inline .gitlab-ci.yml]: This should be "crystal".

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -113,6 +144,19 @@ agent_base_image:
...
+  stage: docker
+  image: docker:19.03.1
+  services:
+    - docker:19.03.1-dind
+  script:
+    - docker login -u gitlab-ci-token -p $CI_BUILD_TOKEN ${CI_REGISTRY}
+    - docker build -t ${host_client_image} --build-arg CI_REGISTRY=${CI_REGISTRY} -f micro_ros_setup/docker/Dockerfile.host_client .
+    - docker push ${host_client_image}
+  only:
+    - scheduled
```
Comment (iluetkeb) [inline .gitlab-ci.yml]: Should by "schedules" -- I also made this mistake in some earlier versions, but current crystal should be correct.

**Comment by BorjaOuterelo** [comment]
@iluetkeb I think it's ready to merge. CI added and working.

**Pull request merged**